### PR TITLE
[TS-162] Brokerage sign-up (link alpaca account to tradeshare account)

### PR DESCRIPTION
**Pull request by tungleu**
Sign up will redirect to brokerage signup to link their alpaca account. 
Note: instead of using `localhost` please use `127.0.0.1` instead
ex: `http://127.0.0.1:8081/dashboard`

### CODE REVIEW

**Comment by Ming424** [comment]
SonarCloud report 
image

**Comment by GadoliniumPotassium** [comment]
From what I read of the changes, good job. I will give it a more thorough test later on. However it looks really good.

**Review by Ming424 — APPROVED** [review]
lgtm, once merged, I will implement api call into hook file

**Review by GadoliniumPotassium — DISMISSED** [review]

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 VUE_APP_BACKEND_HOST=https://tradeshare-backend-for-dev.herokuapp.com/api/v1
-#VUE_APP_BACKEND_HOST=https://tradeshare-backend-for-test.herokuapp.com/api/v1
\ No newline at end of file
+#VUE_APP_BACKEND_HOST=https://tradeshare-backend-for-test.herokuapp.com/api/v1
+VUE_APP_ALPACA_INTEGRATION_URL=https://app.alpaca.markets/oauth/authorize?response_type=code&client_id=1b32c84697cc9bc9c95f646e487c02e7&redirect_uri=http://127.0.0.1:8081/confirm
```
Comment (Ming424) [inline .env:3]: should we git ignore .env file?

**Review by Ming424 — APPROVED** [review]
image
Looks great! alpaca token is there

**Comment by Ming424** [comment]
Steps:
-Have alpaca login
-Use http://127.0.0.1:8081
-Sign up a new
-Step will be shown
-Application - Local Storage: check if token is there

**Review by kennguyen0303 — APPROVED** [review]
Big front-end task. It is working for me, yet making sure you open a task/bug for revamping the front-end page.

**Pull request merged**